src/msw/joystick.cpp

```diff
@@ -157,6 +157,7 @@ wxJoystick::wxJoystick(int joystick)
     JOYINFO joyInfo;
     int i, maxsticks;
 
+    m_thread = nullptr;
```

Comment:
```suggestion
    m_thread = NULL;
```

`nullptr` is C++11, wxWidgets is still C++98, so `NULL` must be used.